corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -106,26 +106,44 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
     var showMenu = function (menuResponse) {
         var menuListView = menusUtils.getMenuView(menuResponse);
         var appPreview = FormplayerFrontend.currentUser.displayOptions.singleAppMode;
-        var queryResponse = menuResponse.queryResponse;
         var sidebarEnabled = !appPreview && menusUtils.isSidebarEnabled(menuResponse);
-        if (sidebarEnabled && menuResponse.type === constants.QUERY) {
-            var menuData = menusUtils.getMenuData(menuResponse);
-            menuData["triggerEmptyCaseList"] = true;
-            menuData["sidebarEnabled"] = true;
-            menuData["description"] = menuResponse.description;
-
-            var caseListView = menusUtils.getCaseListView(menuResponse);
-            FormplayerFrontend.regions.getRegion('main').show(caseListView(menuData));
-        } else if (menuListView) {
+        if (menuListView && !sidebarEnabled) {
```

Comment:
can this be simplified to just `!sidebarEnabled`

Reply:
the very initial implementation checks `menuListView` isn't null so I just kept it. I also think it makes sense since `getMenuView` can potentially return null.